ipaserver/install/krbinstance.py

```diff
@@ -346,8 +346,9 @@ def __create_host_keytab(self):
         self.move_service_to_host(host_principal)
 
     def setup_pkinit(self):
-        ca_db = certs.CertDB(self.realm, host_name=self.fqdn,
-                                subject_base=self.subject_base)
+        ca_db = certs.CertDB(self.realm, paths.HTTPD_ALIAS_DIR,
+                             host_name=self.fqdn,
+                             subject_base=self.subject_base)
```

Comment:
Rather than using `/etc/httpd/alias` here, please move `install_pem_from_p12` out of `CertDB`.